specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -463,6 +473,14 @@ def verify_cell_proof_batch(row_commitments_bytes: Sequence[Bytes48],
     Public method.
     """
     assert len(cells_bytes) == len(proofs_bytes) == len(row_indices) == len(column_indices)
+    for commitment_bytes in row_commitments_bytes:
+        assert len(commitment_bytes) == BYTES_PER_COMMITMENT
+    for cell_bytes in cells_bytes:
+        assert len(cell_bytes) == FIELD_ELEMENTS_PER_CELL
+        for field_bytes in cell_bytes:
+            assert len(field_bytes) == BYTES_PER_FIELD_ELEMENT
+    for proof_bytes in proofs_bytes:
+        assert len(proof_bytes) == BYTES_PER_PROOF
 
```

Comment:
Do we also want to check that row_indices and column_indices are within the bounds here?

Reply:
Yes, that's a good idea. Will add that.